scripts/lighthouse/puppeteer-script.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {puppeteer.Browser} browser
+ * @param {{url: string, options: LHCI.CollectCommand.Options}} context
+ */
+module.exports = async (browser, context) => {
+  const url = 'http://localhost:9000/Article?url=https://www.theguardian.com/commentisfree/2020/feb/08/hungary-now-for-the-new-right-what-venezuela-once-was-for-the-left#noads';
```

Comment:
Think you can get this off of context?

`{{url: string, options: LHCI.CollectCommand.Options}} context`

Which will be useful if URLs is multiple in the array in lighthouserc!

Reply:
oh!! great point, I totally missed that. I'll give it a try